src/main/java/lotto/constant/Rank.java

```diff
@@ -0,0 +1,47 @@
+package lotto.constant;
+
+public enum Rank {
+    FIFTH(5, 2, 0, "꽝"),
+    FOURTH(4, 3, 5000, "3개 일치 (5,000원) - "),
+    THIRD(3, 4, 50000, "4개 일치 (50,000원) - "),
+    SECOND(2, 5, 1500000, "5개 일치 (1,500,000원) - "),
+    FIRST(1, 6, 2000000000, "6개 일치 (2,000,000,000원) - ");
+
+    private final int number;
+    private final int matchingNumberCount;
+    private final int prizeMoney;
+    private final String description;
+
+    Rank(int number, int matchingNumberCount, int prizeMoney, String description) {
+        this.number = number;
+        this.matchingNumberCount = matchingNumberCount;
+        this.prizeMoney = prizeMoney;
+        this.description = description;
+    }
+
+    public int getPrizeMoney() {
+        return prizeMoney;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public int getMatchingNumberCount() {
+        return matchingNumberCount;
+    }
+
+    public int getNumber() {
+        return number;
+    }
+
+    public static Rank value(int matchingCount) {
+        if (matchingCount < 3) return FIFTH;
```

Comment:
if 문의 내부가 한 문장일 경우에도 중괄호로 둘러싸는게 컨벤션이오니 다음 링크를 통해 참고하시면 좋을 것 같아요 :)
```suggestion
        if (matchingCount < 3) {
            return FIFTH;
        }
```

```
4.1.1 Braces are used where optional
Braces are used with if, else, for, do and while statements, even when the body is empty or contains only a single statement
```
[구글 자바 가이드 문서](https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used)